onnxruntime/core/providers/webnn/builders/impl/unary_op_builder.cc

```diff
@@ -49,18 +63,30 @@ Status UnaryOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const
   return Status::OK();
 }
 
+// All unary ops except "Not" and "Identity" contain 'consumed_inputs' attribute before opset 6,
+// which is not supported in WebNN EP for now.
```

Comment:
What is the "consumed_inputs" operator attribute? I don't see that attribute in the ONNX spec: https://github.com/onnx/onnx/blob/main/docs/Operators.md#Add.

Btw, I wouldn't worry too much such old opsets in the future, since the first public release supported opset 7.

Reply:
It is an attribute for opset 1: https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Reciprocal-1
Can I just remove this compatibility restriction?

Reply:
I think we can just ignore it.

Reply:
Oh wow, opset 1. Yeah, we don't need to worry about anything earlier than ONNX 1.2.1 (opset 7), because the ONNX spec was still in flux, ORT hadn't released yet (nor were any other ONNX implementations mature), and any models produced back in those early days have been regenerated with newer opsets anyway.